api/src/test/scala/hmda/api/http/institutions/UploadPathsSpec.scala

```diff
@@ -48,12 +51,14 @@ class UploadPathsSpec extends InstitutionHttpApiSpec with UploadPaths {
         s <- fSubmissionsActor
       } yield {
         s ! UpdateSubmissionStatus(SubmissionId("0", "2017", 1), Signed)
-      }
 
-      Thread sleep 100
-      postWithCfpbHeaders("/institutions/0/filings/2017/submissions/1", file) ~> institutionsRoutes ~> check {
-        status mustBe StatusCodes.BadRequest
-        responseAs[ErrorResponse] mustBe ErrorResponse(400, "Submission already exists", "institutions/0/filings/2017/submissions/1")
+        val submission = Await.result((s ? GetSubmissionById(SubmissionId("0", "2017", 1))).mapTo[Submission], 5.seconds)
+        submission.submissionStatus.code mustBe 12
```

Comment:
Similar to my above comment: I think it's much more readable to compare the actual status, rather than the status code.